packages/nx/src/nx-cloud/utilities/url-shorten.ts

```diff
@@ -14,8 +13,10 @@ export async function shortenedCloudUrl(
   );
 
   const version = await getNxCloudVersion(apiUrl);
-
-  if (version && lt(removeVersionModifier(version), '2406.11.5')) {
+  if (
```

Comment:
What if this function fails?

Tbh, I'd wrap this entire function in a `try/catch`.

Reply:
Updated